django_api_forms/fields.py

```diff
@@ -301,3 +301,19 @@ def to_python(self, value) -> typing.Optional[File]:
         f.seek(0)  # Return to start of the file
 
         return f
+
+
+class RRuleField(Field):
+    default_error_messages = {
+        'not_rrule': _('This field needs to be a rrule object!')
+    }
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+    def to_python(self, value: str):
+        # Dateutil is required for RRuleField
+        from dateutil.rrule import rrulestr
+
+        result = rrulestr(value)
```

Comment:
Instead of raising `ValueError` you should raise `ValidationException`. Check other field implementation for inspiration.

Reply:
Just sent a new commit adding the try/except block. Any other changes, just let me know!

Reply:
And i'm not sure if im supposed to write the changes in the Changelog manually, but if thats the case, i can do it!

Reply:
@paimvictor The CHANGELOG is written manually so feel free to add your changes to it. Create a section for next RC.